src/odm_sharing/tools/share.py

```diff
@@ -130,18 +130,9 @@ def get_tables(org_queries: sh.queries.OrgTableQueries) -> Set[str]:
     return result
 
 
-def gen_filename(in_name: str, org: str, table: str, ext: str) -> str:
-    if in_name == table or not table:
-        # this avoids duplicating the table name when both input and output is
-        # CSV
-        return f'{in_name}-{org}.{ext}'
-    else:
-        return f'{in_name}-{org}-{table}.{ext}'
-
-
-def gen_filepath(outdir: str, in_name: str, org: str, table: str, ext: str
+def gen_filepath(outdir: str, schema_name: str, org: str, table: str, ext: str
                  ) -> FilePath:
-    filename = gen_filename(in_name, org, table, ext)
+    filename = f'{schema_name}-{org}-{table}.{ext}'
```

Comment:
how can two files be for the same table? In any case, the input file name is not used since it doesn't make sense when using multiple input files. The schema is always singular, so that makes more sense I think? 

Ex: the user has the schema `labfilter.csv`, which generates the output filenames:
- `labfilter-mylab-measures.csv`
- `labfilter-mylab-samples.csv`

Reply:
deferred this to a potential issue: #59